ddl/ddl_api.go

```diff
@@ -1269,6 +1269,19 @@ func (d *ddl) AlterTable(ctx sessionctx.Context, ident ast.Ident, specs []*ast.A
 		return errRunMultiSchemaChanges
 	}
 
+	is := d.infoHandle.Get()
```

Comment:
Use a function like `checkAlterRunnable` to make the code cleaner.

Reply:
It seems like ddl_api already have func `func (d *ddl) getSchemaAndTableByIdent(ctx sessionctx.Context, tableIdent ast.Ident) (dbInfo *model.DBInfo, t table.Table, err error) `